doc/api/buffer.md

```diff
@@ -508,7 +508,7 @@ added: v5.10.0
 -->
 
 * `size` {Integer} The desired length of the new `Buffer`
-* `fill` {string | Buffer | Integer} A value to pre-fill the new `Buffer` with.
+* `fill` {string|Buffer|Integer} A value to pre-fill the new `Buffer` with.
```

Comment:
I think it would make sense to be specific to keep `Integer` here because it doesn’t really make sense to pass in non-integer `number`s here… the upper-casing is a bit weird but I’d be okay with it, as long as we try to be consistent

(also /cc @ameliavoncat whose awesome PRs also touch these parts of the docs and who might want to keep track of this discussion)